subsys/dfu/Kconfig

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig DFU_CTX
+	bool "Device Firmware Update Handler"
+
+if (DFU_CTX)
+
+config DFU_CTX_MCUBOOT_SUPPORT
+	bool "MCUBoot Update support"
+	default y
+	select FLASH
+	select FLASH_PAGE_LAYOUT
+	select IMG_MANAGER
+	select MPU_ALLOW_FLASH_WRITE
+	select IMG_ERASE_PROGRESSIVELY
+
+config DFU_CTX_MODEM_UPDATE_SUPPORT
```

Comment:
This should also have some selects and be configured to only be valid for chips with modems

Reply:
I don't see what it should select. See PR for chip only check.